test/snyk-from-commandline.spec.ts

```diff
@@ -36,7 +36,7 @@ describe('test calling snyk-to-html from command line', () => {
     expect(stderr).toEqual('');
 
     const cleanedReport = cleanTimestamp(stdout);
-    expect(cleanedReport).not.toContain('<h2 id="overview">Overview</h2>');
```

Comment:
Marked v5 introduced a breaking change to stop this: https://github.com/markedjs/marked/releases/tag/v5.0.0 

It seems we're not depending on it for anything (at least the tests are passing).